runtime/manta/Cargo.toml

```diff
@@ -129,11 +128,9 @@ std = [
 	'frame-system/std',
 	'frame-system-rpc-runtime-api/std',
 	'frame-try-runtime/std',
-	'pallet-assets/std',
 	'pallet-authorship/std',
 	'pallet-balances/std',
 	'pallet-multisig/std',
-	'pallet-proxy/std',
 	'pallet-utility/std',
 	'pallet-transaction-payment-rpc-runtime-api/std',
```

Comment:
seems like no-std is missing for `pallet_scheduler`

Reply:
See L138?